packages/autorest.go/test/acr/azacr/zz_authentication_client.go

```diff
@@ -103,18 +108,23 @@ func (client *AuthenticationClient) exchangeAADAccessTokenForAcrRefreshTokenHand
 //   - options - AuthenticationClientExchangeAcrRefreshTokenForAcrAccessTokenOptions contains the optional parameters for the
 //     AuthenticationClient.ExchangeAcrRefreshTokenForAcrAccessToken method.
 func (client *AuthenticationClient) ExchangeAcrRefreshTokenForAcrAccessToken(ctx context.Context, service string, scope string, refreshToken string, options *AuthenticationClientExchangeAcrRefreshTokenForAcrAccessTokenOptions) (AuthenticationClientExchangeAcrRefreshTokenForAcrAccessTokenResponse, error) {
+	var err error
+	ctx, endSpan := runtime.StartSpan(ctx, "AuthenticationClient.ExchangeAcrRefreshTokenForAcrAccessToken", client.internal.Tracer(), nil)
+	defer func() { endSpan(err) }()
 	req, err := client.exchangeAcrRefreshTokenForAcrAccessTokenCreateRequest(ctx, service, scope, refreshToken, options)
 	if err != nil {
 		return AuthenticationClientExchangeAcrRefreshTokenForAcrAccessTokenResponse{}, err
 	}
-	resp, err := client.internal.Pipeline().Do(req)
+	httpResp, err := client.internal.Pipeline().Do(req)
```

Comment:
```suggestion
	var httpResp *http.Response
	httpResp, err = client.internal.Pipeline().Do(req)
```

Reply:
This is what tripped me up earlier with my option 1/2 thing.  Since `err` is in the same scope it isn't shadowed.  To keep us honest, I've updated CI to check for this.

Reply:
🤦 right, my bad, it's the case in which `:=` combines short variable declaration with reassignment. Good idea, asking a computer to keep an eye on this